### DISCUSSION TITLE
How to run clangd manually to reproduce the same errors as I see in my editor

### DISCUSSION THREAD

**Discussion by anordal** [clangd/clangd#1171] [Q&A] [answered]
clangd 14.0.3 appears to not understand C++17, and I have a minimal reproducer:
https://bugzilla.opensuse.org/show_bug.cgi?id=1199943

I would like to try some options as suggested in the troubleshooting, like `-log=verbose` and `--query-driver=/path/to/mygcc`:
https://clangd.llvm.org/troubleshooting#cant-find-standard-library-headers-map-stdioh-etc

What's the easiest way to just run it (with those arguments) and get the errors, like for debugging purposes? Just running `clangd` in the right directory doesn't reproduce the error (and `clangd filename.cpp` refers to the help text, which refers to clangd.llvm.org whose troubleshooting doesn't give me a hint).

**Comment by HighCommander4** [comment, marked as answer]
You can run `clangd --check=<filename>` as mentioned in https://clangd.llvm.org/guides/system-headers#you-can-build-your-project-but-clangd-is-complaining-about-missing-headers (the rest of that page might be helpful as well).
